setup/setup.sh

```diff
@@ -1,62 +1,100 @@
-USERNAME="student"
+#! /bin/sh
+
+USERNAME=$(whoami)
```

Comment:
I don't think the system username will always match ROL's username, but I guess it's ok for the moment

Reply:
We can't think in terms of ROL.
There will be no ROL, just the local env (minikube) and the remote env (Dev Sandbox).

Reply:
True, I was thinking about the Developer Sandbox, not sure if it's the same login than ROL?

I mean, the problem we will have is how to know what is the Developer Sandbox username to use it also in Minikube. But I think there's no right answer